code/hud/hudtargetbox.cpp

```diff
@@ -1160,7 +1160,7 @@ void HudGaugeTargetBox::renderTargetAsteroid(object *target_objp)
 		case ASTEROID_TYPE_SMALL:
 		case ASTEROID_TYPE_MEDIUM:
 		case ASTEROID_TYPE_LARGE:
-			strcpy_s(hud_name, NOX("asteroid"));
+			strcpy_s(hud_name, XSTR("Asteroid", 431));
```

Comment:
In English, the strings.tbl entry has a lowercase 'a'.  The NOX entry also has a lowercase 'a', so these match and the string should be changed to lowercase.

I was going to say that the hudbrackets.cpp change does not match and we may have a problem, *except* that the "debris" entry 348 is also lowercase, not uppercase as shown in the file.  So change both "asteroid" and "debris" to lowercase.